MaxText/standalone_checkpointer.py

```diff
@@ -69,6 +74,8 @@ def checkpoint_loop(config, state=None):
   else: # Checkpoint was unavailable, state needs to be initialized
     state, _, _ = max_utils.setup_training_state(model, None,
           tx, config, init_rng, mesh, checkpoint_manager)
+  if config.hardware=='cpu': # simulate real checkpoints on CPUs.
```

Comment:
shouldn't we always do this?

Reply:
Sounds fair, will remove this condition.